{"uuid": "68dcecad-bf36-44c5-ad4f-0392845f936e", "children": ["31ffef15-8f94-4ba6-b8d3-2e2f24655686"], "befores": [{"name": "pw_encryption", "status": "broken", "statusDetails": {"message": "TypeError: 'SubRequest' object is not iterable\n", "trace": "  File \"C:\\Users\\Administrator\\AppData\\Local\\Programs\\Python\\Python36-32\\lib\\site-packages\\pluggy\\callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"C:\\Users\\Administrator\\AppData\\Local\\Programs\\Python\\Python36-32\\lib\\site-packages\\_pytest\\fixtures.py\", line 1126, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"C:\\Users\\Administrator\\AppData\\Local\\Programs\\Python\\Python36-32\\lib\\site-packages\\_pytest\\fixtures.py\", line 932, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"C:\\Users\\Administrator\\PycharmProjects\\test\\conftest.py\", line 23, in pw_encryption\n    pw = list(request)\n"}, "start": 1668157855825, "stop": 1668157855825}], "start": 1668157855825, "stop": 1668157855835}